oozie/src/main/java/org/apache/falcon/workflow/engine/OozieWorkflowEngine.java

```diff
@@ -1603,6 +1603,8 @@ public String getWorkflowStatus(String cluster, String jobId) throws FalconExcep
             } else if (jobId.endsWith("-B")) {
                 BundleJob bundle = client.getBundleJobInfo(jobId);
                 return bundle.getStatus().name();
+            } else if (jobId.contains("-C@")) {
```

Comment:
jobId is just a variable name. We are passing bundles, coordinators, workflows and coord actions to this method. Yes. -C@ is applicable only for coord action instance.